blog/zh/blog/2022/08/12/arm-performance-google-aws-azure-with-apisix.md

```diff
@@ -111,12 +131,16 @@ Tau T2A VM 有多种预定义的 VM 形状，每个 VM 最多 48 个 vCPU，每
 
 从测试结果来看，AWS C7g 相比 GCP T2A 和 Azure Dpsv5 拥有更高的性价比。虽然 Azure Dpsv5 相比 GCP T2A 只有 15% 的性能提升， 但是性价比却高了接近一倍。
 
+由于在 Oracle 云免费套餐中，搭载 2 核心 CPU、8 GB 内存的的虚拟机可以免费使用，这意味着如果要获得和上述 GCP T2A 同等的性能，OCI A1 的成本基本为零，性价比简直令人振奋！
+
 ## 总结
 
 AWS 在 2018 年就推出了首款 ARM 架构的处理器 AWS Graviton，比 GCP 提前了大约 4 年进行了基于 ARM 架构的服务器领域的布局，如今 AWS Graviton 处理器已经发展到了第三代。
 
 通过 Apache APISIX 的性能测试结果和性价比分析，我们可以看出 AWS Graviton3 拥有比 GCP T2A 和 Azure Dpsv5 更高的性能和性价比。这于 AWS 在 ARM 服务器领域深耕多年是分不开的。
 
+由于 Oracle 的慷慨，在业务性能允许的情况下，OCI A1 绝对是个非常好的选择，因为这基本上没有成本（你仍然需要为磁盘付费）。
+
```

Comment:
I think this part can also add some links about Oracle
<img width="700" alt="image" src="https://user-images.githubusercontent.com/39793568/196369322-ca59db41-4b4b-4e91-be77-c5656e3a3e37.png">